internal/pkg/platform/model/key/branchKey.go

```diff
@@ -0,0 +1,66 @@
+// Code generated by ent extension "primarykey", DO NOT EDIT.
+
+package key
+
+import (
+	"database/sql/driver"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/keboola/go-client/pkg/keboola"
+)
+
+type BranchKey struct {
+	BranchID keboola.BranchID
+}
```

Comment:
Do tohot commitu som dal uz realny vygenerovany kod - `BranchKey,ConfigurationKey,RowKey`.

Vygenerovany kod je sucastou git repository.

Tak to odporuca `entgo`:
`generated code should always be checked into source control.`
https://entgo.io/docs/ci/#verify-all-generated-files-are-checked-in

Dava to zmysel, ak aktualizujeme libku, tak pekne uvidime ci sa nieco zmenili.

----

Pre Templates API / Buffer Service nemame vygenerovany kod z `Goa` v repe.
To napravim v dalsich PRs. Povodne som nechcel aby nam to pri intenzivnom vyvoji zavadzalo v PRs.
Ale ako je to uz stabilne, tak to ma z uvedeneho dovodu zmysel "zamknut".